lib/assessment.js

```diff
@@ -8,38 +8,57 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(previousValue, currentValue, index, array) {
+  return currentValue > previousValue ? currentValue : previousValue;
 };
 
 // Question 2
 // Write a callback for reduce that produces
 // the sum of the values in the array
-var sum = function sum() {
-
+var sum = function sum(previousValue, currentValue, index, array) {
+  return previousValue + currentValue;
 };
 
 // Question 3
 // Write a function that returns true
 // if invoking the callback for any element
 // in the array returns a truthy value
-var any = function any(array, callback, thisArg) {
 
+// this seems vague, and I don't exactly know
+// what's being asked here, so I'm making a function
+// that uses a for loop to set every item in the array
+// to be 1
+var any = function any(array, callback, thisArg) {
+  for (var i = 0; i < array; i++) {
+    callback(array[i], i, array) = 1;
```

Comment:
This line is trouble! Line 33 here is trying to assign to the *return value* of `callback(array[i], i, array)`, which we know is some primitive value that can be coerced into a boolean. Assigning to a value is syntactically incorrect!

For the goal you stated, a correct line 33 would read:

`array[i] = 1;`

Also, on line 32, you are testing `i`, a variable containing a number, against `array`, which presumably contains an array. I think what you meant as your loop condition was `i < array.length`.

1/4 credit.